### ISSUE TITLE
BigQuery .exist method in new version 0.28.0

### ISSUE THREAD

**Issue by YvesJacquot** [GoogleCloudPlatform/google-cloud-python#4582]
Hello,

I used to use the following condition in version 0.26.0 of google-cloud-bigquery:
`if dataset.table(my_table_name).exists():
`      

I get the following error message when running it with the newer version 0.28.0:

> AttributeError: 'TableReference' object has no attribute 'exists' 

What is the equivalent of the .exist() method in version 0.28.0 ?

Environment: Python 2.7.10, macOS Sierra 10.12.6, google-cloud 0.31.0

Many thanks

**Comment by tswast** [comment]
That's correct. We removed this method, since all it was doing was an API call to get a table and then discarding the response. The equivalent in 0.28.0 is to call `get_table` and catch the `NotFound` error.

https://github.com/GoogleCloudPlatform/google-cloud-python/blob/fbe4e243d8cf03d4ffc60c7b19e86a0166e44514/bigquery/tests/system.py#L1517-L1523

I see that I overlooked this method when creating the [migration guide](https://cloud.google.com/bigquery/docs/python-client-migration#table_code_changes). I'll leave this issue open until we publish a sample for this method.
